tests/mysql/test-mysql.cpp

```diff
@@ -24,6 +24,16 @@
 std::string connectString;
 backend_factory const &backEnd = *soci::factory_mysql();
 
+TEST_CASE("MySQL connection parameters", "[mysql][connection_parameters]")
+{
+    soci::connection_parameters myparam("mysql", "dbname=test user=root password='test123'");
```

Comment:
It would be nice to test quoted values with spaces in them as this uses a different part of the code.